src/utils/api.ts

```diff
@@ -0,0 +1,68 @@
+import { END_POINT } from "./constants";
+import { FolderData, LinkData, ShardData } from "./type";
+
+interface IData<T = string> {
```

Comment:
최근 인터페이스를 명시할때 앞에 I를 붙혀 작성하는 것은 기피되는 것으로 알고 있습니다.
또안 IData 인터페이스는 응답 타입으로 보여집니다.
아래와 같이 제안드립니다.

```suggestion
type BaseResponse<T = unknown> = { data: T }
```